utils/systemd/daos-server.service

```diff
@@ -1,10 +1,16 @@
 [Unit]
 Description=DAOS Server
-#OnFailure=<mediation services>
+Wants=network-online.target
+After=network-online.target
 
 [Service]
-Restart=always
-#Environment=DAOS_PATH=/usr/lib/daos
-ExecStart=/usr/bin/orterun -np 1 -c 1 --hostfile hostfile --enable-recovery --report-uri /tmp/urifile daos_server
+Type=simple
+RuntimeDirectory=daos_server
+RuntimeDirectoryMode=0755
+#Only specify if you are running out of your development Tree
+#Also comment out the default ExecStart line
```

Comment:
do you also want this line in agent.service file? seems it applies to both

Reply:
You are correct the comment about commenting out the default line should be in the agent file as well. I'll see if I can maybe edit it in the UI here.